### PR TITLE
SKYNET-2284: Add pypicloud dockerfile, helm chart and ncbuild

### PR DESCRIPTION
**Pull request by nightfury1204**
test build: https://jenkins.newscred.com/job/newscred-pypicloud-build/2/console

### CODE REVIEW

**Comment by pratikjoy7** [comment]
Since there will be only one chart for this repo, should we keep the chart under `pypicloud-chart` instead of `charts/pypicloud/*`?

**Comment by nightfury1204** [comment]
> Since there will be only one chart for this repo, should we keep the chart under `pypicloud-chart` instead of `charts/pypicloud/*`?

normally chart are kept in the `charts` folder and folder name is same as the chart name, that's why i followed that.

File: charts/pypicloud/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Since we manage which version of app should be deployed through CD,
+# so this app version has no significance in here
+appVersion: latest
+
+keywords:
+- pypicloud
+home: https://github.com/newscred/configuration-management
+sources:
+- https://github.com/newscred/configuration-management/kubernetes/charts/pypicloud
```
Comment (pratikjoy7) [inline charts/pypicloud/Chart.yaml:28]: Update these values?

File: charts/pypicloud/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+              subPath: nginx.conf
+              readOnly: true
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 80
+            initialDelaySeconds: {{ .Values.appConfig.probe.initialDelay }}
+            periodSeconds: {{ .Values.appConfig.probe.interval }}
+            timeoutSeconds: {{ .Values.appConfig.probe.timeout }}
+            failureThreshold: {{ .Values.appConfig.probe.failureThreshold }}
```
Comment (pratikjoy7) [inline charts/pypicloud/templates/deployment.yaml:49]: Add resource limit for the nginx container as well?

File: charts/pypicloud/Chart.yaml (hunk 1 above)
Comment (nightfury1204) [reply charts/pypicloud/Chart.yaml]: Addresed

File: charts/pypicloud/templates/deployment.yaml (hunk 2 above)
Comment (nightfury1204) [reply charts/pypicloud/templates/deployment.yaml:49]: Added

**Review by pratikjoy7 — APPROVED** [review]

**Pull request merged**